include/solvers/cahn_hilliard.h

```diff
@@ -402,11 +402,24 @@ class CahnHilliard : public AuxiliaryPhysics<dim, GlobalVectorType>
   calculate_phase_statistics();
 
   /**
-   * @brief Writes the phase integral to an output file
+   * @brief Writes the phase statistics to an output file
    */
   void
   write_phase_statistics();
 
+  /**
+   *
+   * @return A pointer to an array of double containing the energies. First element is bulk energy, second is interface energy and third is total energy
```

Comment:
This is not a return function. It is void. Does it make sense to use return here?